packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -276,8 +276,8 @@ describeCompat("stashed ops", "NoCompat", (getTestObjectProvider, apis) => {
 			const directory = await d.getSharedObject<SharedDirectory>(directoryId);
 			directory.set(testKey, testValue);
 			const string = await d.getSharedObject<SharedString>(stringId);
-			const collection = string.getIntervalCollection(collectionId);
-			collection.add({ start: testStart, end: testEnd });
+			// const collection = string.getIntervalCollection(collectionId);
```

Comment:
Should we leave a comment here indicating that we need to re-enable theses tests related to the interval collection, as part of the workaround for [AB#7145](https://dev.azure.com/fluidframework/internal/_workitems/edit/7145)?

Reply:
```suggestion
			// todo re-enable after AB#7145
			// const collection = string.getIntervalCollection(collectionId);
```